src/main/java/calculator/parser/SentenceParser.java

```diff
@@ -0,0 +1,42 @@
+package calculator.parser;
+
+import calculator.Operand;
+import calculator.Operator;
+
+public abstract class SentenceParser<T> {
+    public Operand<T> parse(String sentence){
+        String[] splitted = sentence.split(" ");
+        Operand<T> firstOperand = operandFromValue(fromString(splitted[0]));
+        Operand<T> lastOperand = firstOperand;
+        Operator<T> lastOperator = operatorFromSign(Operator.Sign.fromSignStr(splitted[1]));
+        boolean isOperand = true;
+        connect(lastOperand, lastOperator);
+        for(int i=2;i<splitted.length;i++){
+            String s = splitted[i];
+            if(isOperand){
+                lastOperand = operandFromValue(fromString(s));
+                connect(lastOperator, lastOperand);
+                isOperand = !isOperand;
+                continue;
+            }
+            lastOperator = new Operator<>(s);
+            connect(lastOperand, lastOperator);
+            isOperand = !isOperand;
```

Comment:
```suggestion
            isOperand = true;
```
마찬가지로 위에서 true의 경우 조건문 내에서 continue로 반복문 초기로 돌아갈거라
현재 위치의 isOperand는 false으로 정의되 있을 것 같아 !isOperand는 true로 수정하여 값을 변경하는 방식으로 정의하는게
가독성이 있을 것 같습니다.

Reply:
변경하였습니다!